examples/cross-contract-high-level/tests/general.rs

```diff
@@ -0,0 +1,64 @@
+use near_sdk_sim::{
+    call, deploy, init_simulator, to_yocto, ContractAccount, UserAccount, DEFAULT_GAS,
+    STORAGE_AMOUNT,
+};
+extern crate cross_contract_high_level;
+use cross_contract_high_level::CrossContractContract;
+
+near_sdk_sim::lazy_static::lazy_static! {
+    static ref TOKEN_WASM_BYTES: &'static [u8] = include_bytes!("../res/cross_contract_high_level.wasm").as_ref();
+}
+
+fn init(
+    initial_balance: u128,
+) -> (UserAccount, ContractAccount<CrossContractContract>, UserAccount) {
+    let master_account = init_simulator(None);
+    let contract_account =
+        deploy!(CrossContractContract, "contract", &TOKEN_WASM_BYTES, master_account);
```

Comment:
Why does this version use positional arguments instead of named? I don't find it useful to offer these alternatives. It will be easier to maintain & explain if we only have one way to pass these arguments, and I think we should go with the named arguments as used in the FungibleToken tests.

Reply:
The reason was that all I had at first was positional and then I added the named arguments and this didn't get updated.  Underneath the main case for the a macro is all positional arguments so this is one of them that doesn't take gas/deposit. I agree that we should change this.